package.json

```diff
@@ -68,13 +68,25 @@
       ]
     },
     "mac": {
+      "fileAssociations": {
+        "ext": "slp",
+        "name": "slp",
+        "description": "Slippi File Format",
+        "icon": "./resources/icon.icns"
+      },
       "category": "your.app.category.type",
       "target": [
         "zip",
         "dmg"
       ]
     },
     "win": {
+      "fileAssociations": {
```

Comment:
I think potentially this could be at the root level to prevent needing to re-define for each OS? Wonder if you can just overwrite the `ico` and `icns` icon at the OS level

Reply:
Made it root level, but overwriting the icon requires all the data. Luckily we only have to do that for mac I think